source/billing/credit-card.rst

```diff
@@ -0,0 +1,119 @@
+###################
+Credit Card payment
+###################
+
+.. note::
+   Only users with the Project Admin role will be able to view and modify the
+   credit card panel
```

Comment:
Only users with the Project Admin role will be able to view and modify the billing details